### PR TITLE
IA scanner bulk image upload

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: calico-cloud/image-assurance/scan-image-registries.mdx [hunk 1]
```diff
@@ -134,6 +136,23 @@ You can scan images and report results back to {{prodname}}, or scan images loca
...
+  "images": [
+    "ubuntu:latest",
+    "alpine:latest"
+  ]
+}
+```
+
+```shell
+./tigera-scanner scan --input_file images.json --output_file results.json
+```
```
Comment (stevegaossou) [inline calico-cloud/image-assurance/scan-image-registries.mdx:154]: Should we also include an example somewhere to demonstrate scanning multiple images using the command-line arguments method of specifying multiple images?

File: cut-new-version.sh [hunk 2]
```diff
@@ -5,7 +5,7 @@ version=$2
 
 echo 'Cutting a new version...'
 
-docusaurus docs:version:$product $version
+npx docusaurus docs:version:$product $version
```
Comment (stevegaossou) [inline cut-new-version.sh]: Was this change intended? 

I'm not sure (I'm completely unfamiliar with the new docs tooling), but just wanted to check.

File: calico-cloud/image-assurance/scan-image-registries.mdx (hunk 1 above)
Comment (stevegaossou) [inline calico-cloud/image-assurance/scan-image-registries.mdx]: Should we also include an example somewhere to demonstrate scanning multiple images using the command-line arguments method of specifying multiple images?

File: calico-cloud/image-assurance/scan-image-registries.mdx [hunk 3]
```diff
@@ -153,6 +172,8 @@ Key names must match the full name of arguments passed to the CLI. The configura
 | --warn_threshold        | -w        | CC_WARN_THRESHOLD        | CVSS threshold for Warn scan results. Range from 0.0 - 10.0.                                                                |
 | --fail_threshold        | -f        | CC_FAIL_THRESHOLD        | CVSS threshold for Fail scan results. Range from 0.0 - 10.0.                                                                |
 | --vulnerability_db_path | -p        | CC_VULNERABILITY_DB_PATH | Path to a folder to store vulnerability data (defaults to `$XDG_CACHE_HOME`; if it is not set, defaults to `$HOME/.cache`). |
+| --input_file            | -i        | CC_INPUT_FILE            | Input file containing image URLs.                                                                                           |
```
Comment (stevegaossou) [inline calico-cloud/image-assurance/scan-image-registries.mdx]: I think we should have some mention in the [export scan results section](https://github.com/tigera/docs/blob/main/calico-cloud/image-assurance/scan-image-registries.mdx#export-scan-results) that there is an option to export a batch of images and that file can then be fed into the CLI scanner using this input option.

File: cut-new-version.sh (hunk 2 above)
Comment (ghost) [reply cut-new-version.sh]: Yes it's intended. Npx runs node binaries. Instead of devs having different versions on docusaurus installed on their local machine, npx ensures a specified version is used.
